src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+html {
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
```

Comment:
This should belong to specific class related to link for example. html tag should not contain such styles